packages/block-editor/src/components/block-list/index.js

```diff
@@ -40,13 +40,15 @@ import {
 	BlockEditContextProvider,
 	DEFAULT_BLOCK_EDIT_CONTEXT,
 } from '../block-edit/context';
+import { useBlockCommands } from '../use-block-commands';
 
 const elementContext = createContext();
 
 export const IntersectionObserver = createContext();
 const pendingBlockVisibilityUpdatesPerRegistry = new WeakMap();
 
 function Root( { className, ...settings } ) {
+	useBlockCommands();
```

Comment:
It's not clear to me whether this should be called here. What happens when there are multiple block editors in the page. (Previews are a block editor for instance)

Reply:
Yes. I have the same doubt. Where would you say a better place to register these would be?

Reply:
Probably top level in edit-post and edit-site